src/dev/mod.ts

```diff
@@ -78,7 +80,10 @@ export async function collect(directory: string): Promise<Manifest> {
       }
       if (entry.isFile) {
         const ext = extname(entry.name);
-        if (![".tsx", ".jsx", ".ts", ".js"].includes(ext)) continue;
+        if (
+          ![".tsx", ".jsx", ".ts", ".js"].includes(ext) ||
+          (entry.name.match(TEST_FILE_PATTERN) !== null)
```

Comment:
nitpick: use regex.test instead of string.match

Reply:
Not nitpicking at all, `.test` makes much more sense! Thank you for the suggestion @crowlKats!